tests/integration/trigger-file-select-test.js

```diff
@@ -0,0 +1,57 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import { selectFiles } from 'ember-native-dom-helpers';
+
+moduleForComponent('selectFiles', 'Integration | Test Helper | selectFiles', {
+  integration: true
+});
+
+test('It provids a file under the events target', function(assert) {
+  assert.expect(3);
+  this.onChange = (e) => {
+    assert.equal(e.target.files.length, 1, 'file is present');
+    assert.equal(e.target.files[0].type, 'plain/text', 'file has the set type');
+    assert.equal(e.target.files[0].size, 7, 'file has the set size');
+  };
+
+  this.render(hbs`<input class="target-element" type="file" onchange={{onChange}} />`);
+  selectFiles('.target-element', new Blob(['texters'], { type: 'plain/text' }));
+});
+
+test('Ability to handle multiple files', function(assert) {
+  assert.expect(5);
+  this.onChange = (e) => {
```

Comment:
I'd use `assert` from https://github.com/babel/ember-cli-babel#debug-macros, and to test it you can use https://github.com/workmanw/ember-qunit-assert-helpers#emberassert-assertions.

I think I have some example in Ember Power Select: https://github.com/cibernox/ember-power-select/blob/2952c7e3505231226375d876f8d96b6815069c02/tests/integration/components/power-select/assertions-and-deprecations-test.js#L15-L20